grader/grader.py

```diff
@@ -212,6 +212,9 @@ def _set_applied(self, person):
             person.applied = 'N'
             person.napplied = 0
             return
+        except IndexError:
+            person.napplied = 0
```

Comment:
This should be `1` instead, because that's the most likely (they may have applied more than once too).

Reply:
Never mind, I though this is about missing 'napplied' and not 'applied'. So 0 seems to be the correct default.